index.js

```diff
@@ -29,7 +29,15 @@ ResourceManifestPlugin.prototype.apply = function(compiler) {
 
 function format(assets, path) {
   return `
-ui_page "${path}/index.html"
+  name 'resource name'
+  fx_version 'cerulean'
+  games { 'gta5', 'rdr3 }
+  
+  shared_scripts { 'config.lua' }
```

Comment:
Any chance you could remove shared, client and server scripts please? IIRC, I tried to keep the generated manifest fairly basic and not tied any project's structure.